### DISCUSSION TITLE
Heads-up: planning on doing Restraint Release 0.4.0

### DISCUSSION THREAD

**Discussion by cbouchar** [restraint-harness/restraint#238] [General]
Planning to do Restraint release 0.4.0.  Any comments?  If I don't here from anyone by Thursday, I will proceed.

**Comment by StykMartin** [comment]
LGTM 👍🏻

**Reply by cbouchar** [reply]
Thank you

**Reply by cbouchar** [reply]
I reached out into the grapevine and learned Rawhide/F35 issues are related to https://github.com/actions/virtual-environments/issues/3812 due to incompatible with glibc version.
